Airports/EGJJ/Ownership.txt

```diff
@@ -14,32 +14,32 @@ SECTOR:Only JJAPP:0:0
 OWNER:JJR:JJA
 
 SECTOR:JJTWR:0:2000
-OWNER:JJT:JJR:JJA:J:JS:LWH:LWX:LW:LWN:LWC:LWS:L
-ALTOWNER:Observing LON_W_CTR:JJT:JJA:J:JS:O:LM:LWH:LWX:LW:LWN:LWC:LWS:L
-ALTOWNER:Observing London FIR:O:LM:JJT:JJA:J:JS:LWH:LWX:LW:LWN:LWC:LWS:L
-ALTOWNER:Mentoring TWR/APP:JJTM:JJT:JJAM:JJA:JM:J:JS:LWH:LWX:LW:LWN:LWC:LWS:L
+OWNER:JJT:JJR:JJA:J:JS:LWH:LWX:LW:LWN:LWS:L
+ALTOWNER:Observing LON_W_CTR:JJT:JJR:JJA:J:JS:O:LM:LWH:LWX:LW:LWN:LWS:L
+ALTOWNER:Observing London FIR:O:LM:JJT:JJR:JJA:J:JS:LWH:LWX:LW:LWN:LWS:L
+ALTOWNER:Mentoring TWR/APP:JJTM:JJT:JJR:JJAM:JJA:JM:J:JS:LWH:LWX:LW:LWN:LWS:L
 BORDER:JJTWR
 DEPAPT:EGJJ
 ARRAPT:EGJJ
 
 SECTOR:JJAPP-M:0:3000
-OWNER:JJR:JJA:J:JS:LW:LWN:LWS:L
-ALTOWNER:Observing LON_W_CTR:JJA:J:O:LM:LW:LWN:LWS:L
-ALTOWNER:Observing London FIR:O:LM:JJA:J:LW:LWN:LWS:L
-ALTOWNER:Mentoring TWR/APP:JJAM:JJA:JM:J:LW:LWN:LWS:L
+OWNER:JJR:JJA:J:JS:LWH:LWX:LW:LWN:LWS:L
+ALTOWNER:Observing LON_W_CTR:JJR:JJA:J:LWH:LWX:O:LM:LW:LWN:LWS:L
+ALTOWNER:Observing London FIR:O:LM:JJR:JJA:J:LWH:LWX:LW:LWN:LWS:L:L
```

Comment:
:L repeated

Reply:
I thought :L was an emoji that hadn't processed correctly and you were making me work to find the error 😄 

```suggestion
ALTOWNER:Observing London FIR:O:LM:JJR:JJA:J:LWH:LWX:LW:LWN:LWS:L
```